src/coreComponents/codingUtilities/static_if.hpp

```diff
@@ -28,14 +28,14 @@ template<bool CONDITION>
 struct static_if_wrapper
 {
   template<typename LAMBDA_BODY>
-  constexpr inline static void if_function(LAMBDA_BODY&&) { }
+  constexpr inline static void ifFunction(LAMBDA_BODY&&) { }
```

Comment:
@corbett5 @klevzoff @wrtobin Are we still using these?

Reply:
@rrsettgast so I'm starting the merge! (submodules come first)
Anything can be modified in a future PR if needed.